docs/src/inputs/input/devSandbox/devSandbox.jsx

```diff
@@ -0,0 +1,565 @@
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onBlur</code> event from parent.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnBlur').default}
+                >
+                    <ExampleInputOnBlur />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onChange Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onChange</code> event from parent. The <code>value</code> prop is required along with this handler.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnChange').default}
+                >
+                    <ExampleInputOnChange />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onClick Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onClick</code> event from parent.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnClick').default}
+                >
+                    <ExampleInputOnClick />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onFocus Event Handler
+                    </Heading>
+
+                    <Typography
+                        variant="body1"
+                    >
+                        Can handle an <code>onFocus</code> event from parent.
+                    </Typography>
+                </MarkdownContainer>
+
+                <Example
+                    rawCode={require('!!raw-loader!../examples/exampleInputOnFocus').default}
+                >
+                    <ExampleInputOnFocus />
+                </Example>
+
+                <MarkdownContainer>
+                    <Heading
+                        anchorLink="disabled"
+                        variant="h2"
+                    >
+                        onKeyDown Event Handler
+                    </Heading>
```

Comment:
`anchorLink` should be something like `"onkeydown"`

Reply:
Fixed please check this commit https://github.com/saddlebackdev/react-cm-ui/pull/353/commits/64c4ee036e2f80b3878becd191027efa8274376b

Reply:
Looks good.  Resolving.